tests/input_set.py

```diff
@@ -135,6 +135,116 @@ def create_geometry(self):
         super(PinCellInput, self).create_geometry()
 
 
+class GridInput(InputSet):
+    """Sets up a simple grid problem using explicit plane"""
+
+    def create_materials(self):
+        """Instantiate C5G7 Materials."""
+        self.materials = \
+            openmoc.materialize.load_from_hdf5(filename='c5g7-mgxs.h5',
+                                               directory='../../sample-input/')
+
+    def create_geometry(self):
+        """Instantiate a 3x3 grid Geometry."""
+
+        # Create the planes that bound the cell and geometry
+        xplanes = [None] * 4
```

Comment:
Ellaborate on how that would work. My inclination is to tell you no because doing something like ```xplanes = [openMOC.Xplane(...)] * 4``` would just create 4 pointers to the same data.